src/v2/routes/putStore.ts

```diff
@@ -36,10 +36,10 @@ export const putStoreRouter = (appState: AppState): Router => {
       timestamp
     })
 
-    // Send response
-    res.status(201).json(
-      makeApiResponse<PutStoreResponse>({ timestamp })
-    )
+    // Response:
+
```

Comment:
Extra newline here?

Reply:
It was to make it more consistent with the other route files. The made up convention here is that a comment on its own line (not contiguous to code) and ending with a `:` is a sort of section label. Purely my own style